android/geowidget/src/main/java/org/smartregister/fhircore/geowidget/screens/GeoWidgetFragment.kt

```diff
@@ -260,11 +307,11 @@ class GeoWidgetFragment : Fragment() {
         mapView.onSaveInstanceState(outState)
     }
 
-    fun addLocationToMap(location: GeoWidgetLocation) {
+    fun addLocationToMap(location: Feature) {
         geoWidgetViewModel.addLocationToMap(location)
     }
 
-    fun addLocationsToMap(locations: Set<GeoWidgetLocation>) {
+    fun addLocationsToMap(locations: Set<Feature>) {
```

Comment:
Move single code block to the site where this single liner function is called only if it's used in one place.

Reply:
This method is being called from `GeoWIdgetLauncherFragment` and acting as a bridge between `GeoWidgetViewModel` and `GeoWIdgetLauncherFragment`